repos/fdbt-site/src/pages/viewPassengerTypes.tsx

```diff
@@ -70,7 +70,7 @@ const IndividualPassengerTypes = ({ passengerTypes }: { passengerTypes: Passenge
                         className="govuk-grid-column-one-half govuk-!-margin-bottom-5"
                     >
                         <div className="card">
-                            <div className="card__body">
+                            <div className="card__body individual-passenger-type">
```

Comment:
What does this do?

Reply:
Allows us to set the height for this specific setting's card in `repos/fdbt-site/src/design/modules/_cards.scss`

Reply:
👍